inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp

```diff
@@ -305,7 +305,7 @@ InferenceEngine::Blob::Ptr MKLDNNPlugin::MKLDNNInferRequest::GetBlob(const std::
                                                                                                          desc.getShape().getRank()))
                                                                    : MemoryDescUtils::convertToTensorDesc(desc);
                         const auto &tensorDesc = data->getTensorDesc();
-                        if (expectedTensorDesc.getPrecision() != tensorDesc.getPrecision()) {
+                        if (expectedTensorDesc.getPrecision() != normalizeToSupportedPrecision(tensorDesc.getPrecision())) {
```

Comment:
@jiwaszki Could you clarify which tests or use-cases require this change?

Reply:
This one test fails without this change: ```tests/test_onnx/test_zoo_models.py::OnnxBackendModelExecutionTest::test_onnx_model_zoo_text_machine_comprehension_bert_squad_model_bertsquad_10_bertsquad_10_bertsquad10_cpu```
https://dev.azure.com/openvinoci/dldt/_build/results?buildId=197937&view=logs&j=8e6fb299-7deb-52f1-2f4a-df91230aba00&t=f812e137-0344-51cb-e8b0-0ab752070263&l=6178